doc-content/kogito-docs/src/main/asciidoc/drl/con_drl-rule-units.adoc

```diff
@@ -46,7 +46,7 @@ rule "Using a rule unit with a declared type"
   then   // Insert Mark, who is a customer of James.
     Person mark = new Person();
     mark.setName( "Mark" );
-    person.add( mark );
+    person.append( mark );
```

Comment:
On a DataStream you can do 'append' not 'add'.

Reply:
Done.